### DISCUSSION TITLE
Screen sharing under high resolution display not working

### DISCUSSION THREAD

**Discussion by rrentea** [hyprwm/Hyprland#7187] [Q&A]
Hi, i have been using hyprland for a few weeks and had some trouble with screen sharing. I followed the instructions [here](https://wiki.hyprland.org/Useful-Utilities/Screen-Sharing/) but it's still not working properly.

I found out recently that it was not working because of my high resolution display (2560x1440). As a workaround when i need screen sharing i switch to a lower resolution and it works fine on firefox and chromium (slack still doesn't work). 

What might be the issue?
Thanks!

**Comment by solomonton** [comment]
Have you tried following all the points in these [instructions](https://gist.github.com/brunoanc/2dea6ddf6974ba4e5d26c3139ffb7580) ?
